### PR TITLE
update rai envs in rai components

### PR DESCRIPTION
**Pull request by qusongms**
update rai envs in rai components:
* responsibleai-tabular/versions/1 -> responsibleai-tabular/versions/2
* responsibleai-text/versions/1 -> responsibleai-text/versions/2

### CODE REVIEW

File: assets/responsibleai/vision/components/rai_vision_insights/spec.yaml [hunk 1]
```diff
@@ -116,7 +116,7 @@ outputs:
     type: path
     description: Json file to which UX is serialized to for viewing in static AzureML Studio UI
 code: ../src
-environment: azureml://registries/azureml/environments/responsibleai-vision/versions/1
+environment: azureml://registries/azureml/environments/responsibleai-vision/versions/2
```
Comment (imatiach-msft) [inline assets/responsibleai/vision/components/rai_vision_insights/spec.yaml:119]: not completely sure if we want to do this yet, given automl is still on older version of pytorch

**Review by imatiach-msft — DISMISSED** [review]

File: assets/responsibleai/vision/components/rai_vision_insights/spec.yaml (hunk 1 above)
Comment (qusongms) [reply assets/responsibleai/vision/components/rai_vision_insights/spec.yaml]: revert to /responsibleai-vision/versions/1

File: assets/responsibleai/text/jobs/fetch_dbpedia_model.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ outputs:
   model_output_path:
     type: path
 code: ./src_dbpedia/
-environment: azureml://registries/azureml/environments/responsibleai-text/versions/1
+environment: azureml://registries/azureml/environments/responsibleai-text/versions/2
```
Comment (Advitya17) [inline assets/responsibleai/text/jobs/fetch_dbpedia_model.yml:19]: trailing space?

**Review by Advitya17 — APPROVED** [review]

File: assets/responsibleai/text/jobs/fetch_dbpedia_model.yml (hunk 2 above)
Comment (qusongms) [reply assets/responsibleai/text/jobs/fetch_dbpedia_model.yml]: what a catch! corrected.

**Review by Advitya17 — APPROVED** [review]

**Pull request merged**